raft/raft.go

```diff
@@ -55,12 +55,14 @@ func (st StateType) String() string {
 type Config struct {
 	// ID is the identity of the local raft. ID cannot be 0.
 	ID uint64
-	// Peers contains the IDs of all nodes (including self) in
+
+	// peers contains the IDs of all nodes (including self) in
 	// the raft cluster. It should only be set when starting a new
 	// raft cluster.
 	// Restarting raft from previous configuration will panic if
-	// Peers is set.
-	Peers []uint64
+	// peers is set.
+	// peer is private and only used for testing right now.
+	peers []uint64
```

Comment:
I was thinking of moving this out of the Config struct completely as a separate argument to newRaft, but I guess an unexported field works too. LGTM